scripts/llnl/build_src.py

```diff
@@ -52,6 +52,10 @@ def parse_args():
                       dest="verbose",
                       default=False,
                       help="Output logs to screen as well as to files")
+    parser.add_option("-j", "--jobs",
+                      dest="jobs",
+                      default=4,
```

Comment:
I think we should default to just `-j` which will use the max system amount instead of picking an arbitrary amount.

Reply:
agreed. fixed.